crates/bevy_pbr/src/light_probe/mod.rs

```diff
@@ -104,6 +103,7 @@ pub struct LightProbePlugin;
 /// with other engines should be aware of this terminology difference.
 #[derive(Component, Debug, Clone, Copy, Default, Reflect)]
 #[reflect(Component, Default, Debug)]
+#[require(Transform, Visibility)]
 pub struct LightProbe;
```

Comment:
Not directly related to this PR, but should this require `SyncToRenderWorld`? Lights require it, but this isn't really a light itself, and I'm not too sure which components should or shouldn't have the component. The bundle didn't have it, so I didn't add it yet.

Reply:
It doesn't look like it's ever used in the render world, so it doesn't need to be synced